rep-2000.rst

```diff
@@ -709,6 +709,121 @@ Package manager use for dependencies:
+|              |                       |                         |                      |                    | webOS OSE      |
++==============+=======================+=========================+======================+====================+================+
+|    amd64     |   Tier 1 [d][a][s]    |     Tier 1 [a][s]       |    Tier 1 [a][s]     |     Tier 3 [s]     |                |
++--------------+-----------------------+-------------------------+----------------------+--------------------+----------------+
+|    arm64     |   Tier 1 [d][a][s]    |                         |                      |     Tier 3 [s]     |   Tier 3 [s]   |
++--------------+-----------------------+-------------------------+----------------------+--------------------+----------------+
+|    arm32     |     Tier 3 [s]        |                         |                      |     Tier 3 [s]     |   Tier 3 [s]   |
++--------------+-----------------------+-------------------------+----------------------+--------------------+----------------+
+
+The following indicators show what delivery mechanisms are available for each platform.
+
+" [d] " Debian packages will be provided for this platform for packages submitted to the rosdistro.
+
+" [a] " Binary releases are provided as a single archive per platform containing all packages in the Eloquent ROS 2 repos file [6]_.
+
+" [s] " Compilation from source.
+
+Middleware Implementation Support:
+
++--------------------------+---------------------+---------------+-----------------------------+--------------------------------------+
+| Middleware Library       | Middleware Provider | Support Level | Platforms                   | Architectures                        |
++==========================+=====================+===============+=============================+======================================+
+| rmw_fastrtps_cpp         | eProsima Fast-DDS   | Tier 1        | All Platforms               | All Architectures                    |
++--------------------------+---------------------+---------------+-----------------------------+--------------------------------------+
+| rmw_connext_cpp          | RTI Connext         | Tier 1        | All Platforms except Debian | All Architectures except arm64/arm32 |
+|                          |                     |               | and OpenEmbedded            |                                      |
++--------------------------+---------------------+---------------+-----------------------------+--------------------------------------+
+| rmw_cyclonedds_cpp       | Eclipse Cyclone DDS | Tier 1        | All Platforms               | All Architectures                    |
++--------------------------+---------------------+---------------+-----------------------------+--------------------------------------+
+| rmw_fastrtps_dynamic_cpp | eProsima Fast-DDS   | Tier 2        | All Platforms               | All Architectures                    |
```

Comment:
I'll suggest that we change up the ordering of this list a bit.  Essentially first sort on Tier support, then sort on open-source, then sort alphabetically.  That would leave the order of the list like this:

* rmw_cyclonedds_cpp
* rmw_fastrtps_cpp
* rmw_connext_cpp
* rmw_fastrtps_dynamic_cpp

Also, @nuclearsandwich should we add GurumDDS to this list somewhere?

Finally, since we aren't supporting arm32 anywhere, you can remove that exception from the `rmw_connext_cpp` line.

Reply:
c07dad6